<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>1061</width>
     <height>891</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Camera</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>228</width>
       <height>74</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>228</width>
       <height>74</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>90</y>
       <width>228</width>
       <height>74</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>720</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color=rgb(85, 255, 0)</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>IMage Loading Section</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Gallery</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>228</width>
       <height>74</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>228</width>
       <height>74</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>90</y>
       <width>228</width>
       <height>74</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Report</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>321</width>
       <height>741</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>321</width>
       <height>741</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>90</y>
       <width>321</width>
       <height>741</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Disease A</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Disease B</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Disease C</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Exit</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>228</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>690</y>
     <width>228</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>228</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>228</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gallery</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
